### PR TITLE
Add policy for core contributor expanded access

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Here's a draft of a proposed policy and process for giving community members access to private repos or other systems that are not public.

### CODE REVIEW

File: operations/research-and-development/engineering/core-contributor-expanded-access.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+<contributor-name> has been a reliable part of the community for <length-of-time> and is trusted.
+
+Thanks,
+
+<your-name>
+```
+
+2. Wait for the division head and Security Operations to approve via email.
+3. Ask the community member to sign the standard Mattermost NDA (link for process needed).
```
Comment (jwilander) [inline operations/research-and-development/engineering/core-contributor-expanded-access.md:31]: I spoke with Nirosha about the ideal process for this. There's a new tool and process coming soon to support this. I'll add the link once the documentation on that process is ready.

**Review by iyampaul — DISMISSED** [review]

**Review by spirosoik — DISMISSED** [review]
Thanks for this Joram. LGTM

**Review by jordanafung — DISMISSED** [review]

File: operations/research-and-development/engineering/core-contributor-expanded-access.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Core contributor expanded access policy
+
+At times we have community members who are contributing to projects or areas that require access to private repositories or systems that are not typically available to community members. Access can be granted on a case-by-case basis if:
```
Comment (cwarnermm) [inline operations/research-and-development/engineering/core-contributor-expanded-access.md]:
```suggestion
At times we have community members who are contributing to projects or areas that require access to private repositories or systems that are not typically available to community members. Access can be granted on a case-by-case basis when:
```
```suggestion
At times we have community members who are contributing to projects or areas that require access to private repositories or systems that are not typically available to community members. Access can be granted on a case-by-case basis if:
```

File: operations/research-and-development/engineering/core-contributor-expanded-access.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+* The contributor is a trusted, long-standing member of the community.
+* They are not being given access to production systems.
+* The division head for the system being given access to and Security Operations approves.
```
Comment (cwarnermm) [inline operations/research-and-development/engineering/core-contributor-expanded-access.md]:
```suggestion
* The system's division head and Security Operations approves.
```

**Review by cwarnermm — COMMENTED** [review]
Great update, @jwilander! Thank you! I've provided editorial feedback inline.

**Comment by jwilander** [comment]
Thanks @cwarnermm !

**Review by cwarnermm — APPROVED** [review]

**Comment by cwarnermm** [comment]
@jwilander - Feel free to merge when you're ready for this to go live.

**Pull request merged**